packages/amplify-util-mock/src/api/api.ts

```diff
@@ -17,6 +17,9 @@ import { getInvoker } from 'amplify-category-function';
 import { lambdaArnToConfig } from './lambda-arn-to-config';
 import { timeConstrainedInvoker } from '../func';
 
+export const GRAPHQL_API_ENDPOINT_OUTPUT = 'GraphQLAPIEndpointOutput';
+export const GRAPHQL_API_KEY_OUTPUT = 'GraphQLAPIKeyOutput';
+export const MOCK_API_KEY = 'da2-fakeApiId123456';
```

Comment:
although it's slightly tangental to this PR, this const should be used in the other places where we use this value as well: https://github.com/aws-amplify/amplify-cli/search?q=da2-fakeApiId123456